schemas/chronopost/generate.sh

```diff
@@ -0,0 +1,7 @@
+SCHEMAS=./schemas
+LIB_MODULES=./chronopost_lib
+mkdir -p $LIB_MODULES
+find "${LIB_MODULES}" -name "*.py" -exec rm -r {} \;
+touch "${LIB_MODULES}/__init__.py"
+
+generateDS --no-namespace-defs -o "${LIB_MODULES}/services.py" "${SCHEMAS}/ShippingServiceWS.xml"
```

Comment:
I looked at the documentation and for the full integration of the shipping service, you will also need the QuickCost schema to parse the rating responses

https://ws.chronopost.fr/quickcost-cxf/QuickcostServiceWS?wsdl

Reply:
Actually, you might as well generate the schema types for TrackingService since the response for cancel shipment (cancelSkyBill) response schema will be required

https://ws.chronopost.fr/tracking-cxf/TrackingServiceWS?wsdl